src/lib/utils/__tests__/classNames.js

```diff
@@ -0,0 +1,33 @@
+import classNames from '../classNames';
+
+describe('classNames', () => {
+  it('returns filtered class names', () => {
+    const result = classNames(
```

Comment:
We should probably also test with two class names.
Maybe also possibly with class name containing space?

Reply:
Why not. I just wanted to cover all typical cases that can accidentally be passed into it but we test it as well.